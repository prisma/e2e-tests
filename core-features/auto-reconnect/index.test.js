const { client } = require('.')

describe('should test Prisma client and PgBouncer', () => {
  afterAll(async () => {
    await client.$disconnect()
  })

  it('should test auto-reconnect', async () => {
    const data = await client.user.findMany({
      where: {
        id: 'x',
      },
    })
    expect(data).toMatchSnapshot()

    await client.$disconnect()

    const data1 = await client.user.findMany({
      where: {
        id: 'x',
      },
    })
    expect(data1).toMatchSnapshot()
  })
})
