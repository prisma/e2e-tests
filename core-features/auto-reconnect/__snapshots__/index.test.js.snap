// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Prisma client and PgBouncer should test auto-reconnect 1`] = `Array []`;

exports[`should test Prisma client and PgBouncer should test auto-reconnect 2`] = `Array []`;
